src/modules/Hosts/Hosts/Views/MainPage.xaml

```diff
@@ -69,26 +67,24 @@
             </Button>
 
             <StackPanel
-                Padding="0,0,18,0"
                 HorizontalAlignment="Right"
-                Orientation="Horizontal">
+                Orientation="Horizontal"
+                Spacing="8">
```

Comment:
This needs to be 4 to have a smaller gap.

For `DefaultButtonStyle` having 8 would be more appropriate.. but here we are using the `SubtleButtonStyle` which has no visual borders and therefore spacing is less of an issue to feel cramped to cluttered. On the contrary: this feels stretched.

See Snipping Tool as a reference: 
<img width="175" alt="image" src="https://github.com/microsoft/PowerToys/assets/9866362/777fb2c9-d27d-45c3-9ca8-e1b0bb5f2b85">

Reply:
I see. Why don't we use a CommandBar here, then? :)

Reply:
Because it might come with a bunch of other things we do not want.. it's just a few buttons :)? Snipping Tool also uses buttons vs. command bar